packages/core/src/diff/changes/change.ts

```diff
@@ -86,28 +86,42 @@ export const ChangeType = {
   // Directive Usage
   DirectiveUsageUnionMemberAdded: 'DIRECTIVE_USAGE_UNION_MEMBER_ADDED',
   DirectiveUsageUnionMemberRemoved: 'DIRECTIVE_USAGE_UNION_MEMBER_REMOVED',
+  DirectiveUsageUnionMemberChanged: 'DIRECTIVE_USAGE_UNION_MEMBER_CHANGED',
   DirectiveUsageEnumAdded: 'DIRECTIVE_USAGE_ENUM_ADDED',
   DirectiveUsageEnumRemoved: 'DIRECTIVE_USAGE_ENUM_REMOVED',
+  DirectiveUsageEnumChanged: 'DIRECTIVE_USAGE_ENUM_CHANGED',
   DirectiveUsageEnumValueAdded: 'DIRECTIVE_USAGE_ENUM_VALUE_ADDED',
   DirectiveUsageEnumValueRemoved: 'DIRECTIVE_USAGE_ENUM_VALUE_REMOVED',
+  DirectiveArgumentUsageEnumValueAdded: 'DIRECTIVE_USAGE_ARGUMENT_ENUM_VALUE_ADDED',
+  DirectiveArgumentUsageEnumValueRemoved: 'DIRECTIVE_USAGE_ARGUMENT_ENUM_VALUE_REMOVED',
+  DirectiveArgumentUsageEnumValueChanged: 'DIRECTIVE_USAGE_ARGUMENT_ENUM_VALUE_CHANGED',
```

Comment:
All of these `DirectiveUsage<insert-type>Changed` change types were prematurely added by me. We need to turn them into much granular added, removed, and changed types. Just like how it's done for types `DirectiveArgumentUsageEnumValue<Added | Removed | Changed>` here.